### PR TITLE
Add PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by sanmai**
This PR:

-  [ ] Covered by tests
-  [ ] Adds a new feature ...
-  [ ] Fixes #nnnn

(looks just like this)

### CODE REVIEW

**Comment by pimjansen** [comment]
Maybe also add a checkbox to verify that the docs are updated? Since this misses a lot of times

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+This PR:
+
+- [ ] Adds a new feature ...
+- [ ] Covered by tests
+- [ ] Fixes #nnnn
```
Comment (localheinz) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
This PR

* [x]
* [ ]

Follows #.
Related to #.
Fixes #.
```

**Review by localheinz — COMMENTED** [review]
❓

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (sanmai) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: - It seems more obvious what to do with the checkmarks with they have some labels. 
- And I like a dedicated "Covered by tests": saves you from reminding an author that they need to add tests.

What do you think?

Comment (localheinz) [reply .github/PULL_REQUEST_TEMPLATE.md:5]: 👍

**Review by localheinz — APPROVED** [review]

**Comment by localheinz** [comment]
Thank you, @pimjansen and @sanmai!

**Pull request merged**